docs/pages/react/login-methods/email-magic-link.mdx

```diff
@@ -112,7 +112,8 @@ const handleSendMagicLink = (email: string) => {
     },
     {
       onSuccess: () => {
-        // Success - inform user to check their email
+        // onSuccess only fires once the entire flow is done (email magic link + optional MFA).
+        // It still runs even if the final step completes in another tab/window.
```

Comment:
"It still runs even if the final step completes in another tab/window."

added this bit in an attempt to explain a little why we do this